.github/dockerfiles/ov_test/ubuntu_22_04_android/Dockerfile

```diff
@@ -0,0 +1,97 @@
+ARG REGISTRY="docker.io"
+FROM ${REGISTRY}/library/ubuntu:22.04
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+# Install system dependencies for Android testing
+RUN apt update && \
+    apt install \
+        software-properties-common \
+        git \
+        ca-certificates \
+        wget \
+        curl \
+        unzip \
+        build-essential \
+        python3 \
+        python3-pip \
+        openjdk-17-jdk && \
+    # create a stable JAVA_HOME symlink based on the installed JDK \
+    ln -s "$(dirname $(dirname $(readlink -f $(which javac))))" /usr/lib/jvm/default-java && \
+    test -x /usr/lib/jvm/default-java/bin/java && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set Java environment
+ENV JAVA_HOME="/usr/lib/jvm/default-java"
+ENV PATH="$JAVA_HOME/bin:$PATH"
+
+# Enable KVM support (required for Android emulator)
+RUN apt update && \
+    apt install \
+        qemu-kvm \
+        libvirt-daemon-system \
+        libvirt-clients \
+        bridge-utils && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Android SDK and tools
+ENV ANDROID_HOME="/opt/android-sdk"
+ENV ANDROID_SDK_ROOT="$ANDROID_HOME"
+ENV PATH="$ANDROID_HOME/cmdline-tools/latest/bin:$ANDROID_HOME/cmdline-tools/bootstrap/bin:$ANDROID_HOME/platform-tools:$PATH"
+
+# Android configuration
+ENV ANDROID_API_LEVEL=29
+ENV ANDROID_TARGET="google_apis"
+ENV ANDROID_ARCH="x86_64"
+
+RUN mkdir -p ${ANDROID_HOME}/cmdline-tools && \
+    cd /tmp && \
+    wget -q https://dl.google.com/android/repository/commandlinetools-linux-7583922_latest.zip && \
+    unzip -q commandlinetools-linux-7583922_latest.zip && \
+    # place as a bootstrap toolset; then we install "cmdline-tools;latest" via sdkmanager
+    mv cmdline-tools ${ANDROID_HOME}/cmdline-tools/bootstrap && \
+    rm -f commandlinetools-linux-7583922_latest.zip && \
+    mkdir -p /root/.android && touch /root/.android/repositories.cfg
+
+# Accept Android licenses and install SDK components
+RUN yes | sdkmanager --sdk_root=${ANDROID_HOME} --licenses && \
```

Comment:
(not an expert here, but) are we sure we can use this license?

Reply:
I don't think so, although I'm not an expert. Logically, it should not be a problem